packages/replay-next/pages/variables.css

```diff
@@ -975,6 +974,8 @@
   --cm-def: #3172e0;
   --cm-string: #0842a4;
   --cm-number: #18181a;
+  /* --cm-property: ; to add */
+  /* --cm-atom: #fff; to add */
```

Comment:
FWIW~ I _think_ all of the "--cm" variables are no longer used, now that Code Mirror is gone.